subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/dependency_resolution.adoc

```diff
@@ -0,0 +1,104 @@
+= Dependency resolution
+
+This chapter covers the way dependency resolution works _inside_ Gradle.
+After covering how you can declare <<declaring_repositories.adoc#,repositories>> and <<declaring_dependencies.adoc#,dependencies>>, it makes sense to explain how these declarations come together during dependency resolution.
+
+[[sec:how-gradle-downloads-deps]]
+== How Gradle downloads dependencies
```

Comment:
:o: It appears this section doesn't mention how resolution works in a higher level, in particular:

- one `Configuration` = one dependency graph
- artifacts are downloaded lazily

Reply:
Indeed, this new chapter needs work!